test/failures_testsuite/config.yaml

```diff
@@ -10,11 +10,19 @@ s3:
   deploy: True # True > deploy, false > use
   instance:
     farm: 'kristof-farm'
-    size: 10000
-    shards: 4
-    parity: 2
+    size: 1000
+    shards: 1
+    parity: 1
```

Comment:
-  why shards is 1 ? same for the parity?
- I also suggest to make deploy in s3 and s3_red to be False as default  for safety .. in case the person who run the code didn't check the config file

Reply:
- There is no constrains one shards or parity numbers, so I put as minimum as I can
- The default must match what we expect to be done with CI tool, and we expect that it should install a new instance.